modules/library/metadata/pom.xml

```diff
@@ -146,6 +171,37 @@
         <directory>${basedir}/src/main/properties</directory>
       </resource>
     </resources>
+    <plugins>
+      <plugin>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <version>3.2.2</version>
+        <executions>
+          <execution>
+            <id>integration-test</id>
+            <goals>
+              <goal>install</goal>
+              <goal>run</goal>
+              <goal>verify</goal>
+            </goals>
+            <configuration>
+              <goals>
+                <goal>exec:exec</goal>
+              </goals>
+              <verifyHookScript>verify.bsh</verifyHookScript>
```

Comment:
Oh... interesting use of integration tests with beanshell.

Reply:
The choice given by invoker plugin is beanshell or groovy, I chose well.

The real annoying part is having to use `exec:exec` to run java; since use of `exec:java` runs in process and uses the already configured logging environment. To get an independent logging environment `exec:exec` is needed.